modules/network/mainnet/index.ts

```diff
@@ -0,0 +1,9 @@
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { mainnetNetworkData as data } from './data';
+import { mainnetWorkerJobs as workerJobs } from './workers';
+
```

Comment:
missing export

Reply:
actually it's not needed. i'm removing from other networks as well.